### DISCUSSION TITLE
Widgets

### DISCUSSION THREAD

**Discussion by tituscmd** [InfiniTimeOrg/InfiniLink#131] [Ideas]
Are there any ideas on creating iOS widgets for InfiniLink? You could have one for your HRM data with the amount of data shown taken from your last picked sliders. Or maybe one to show how many steps you've taken today? I think that would be interesting. Let me know what you think about this!

**Comment by liamcharger** [comment]
Yes! I have experimented with widgets in the past: widgets that show your heart rate data or step count like you mentioned, and widgets that shows the current state of the battery.

I'll look into it more when I have time, when I've squashed some crashes and bugs in the current TestFlight build ;) and have pushed the app to the App Store.

**Reply by tituscmd** [reply]
That's great! I hope the new update comes to my phone soon.
